lib/poll.js

```diff
@@ -4,19 +4,33 @@ export default class Poll extends EE.EventEmitter {
   constructor (options, frequency) {
     super();
     this.url = 'https://' + options.host + '/Remote/GetActiveTableID';
-    this.opts = {
-      shortTableId: options.account + '/' + options.table
-    };
+    this.shortTableId = options.account + '/' + options.table
+
+    if (options.cacheBust) {
+      this.cacheBust = options.cacheBust;
+    } else {
+      this.cacheBust = fasle;
+    }
     this.table = void 0;
     frequency = frequency || 30;
     this.interval = frequency * 1000;
     this.timeout = void 0;
     this.aborted = false;
     this.checkTable();
   }
+  getCacheBust() {
+    // return Math.random().toString().slice(2);
+    return Math.round(Date.now()/1000);
+  }
   checkTable() {
     var self = this;
-    return ajax(this.url, this.opts).then(function (resp) {
+    const opts = {
+      shortTableId: this.shortTableId
```

Comment:
Don't you need to add an `mluser` and either an `mltoken` or `mlpass` along with this request?

[See `GetActiveTableId` documentation](https://maplarge.com/developer/webservices/getactivetableid)

Reply:
no, because we are using this to access public data